test/test_warp_reduce.cu

```diff
@@ -67,16 +69,15 @@ struct WrapperFunctor
     template <typename T>
     inline __host__ __device__ T operator()(const T &a, const T &b) const
     {
-      if (CUB_IS_DEVICE_CODE)
-      {
-          #if CUB_INCLUDE_DEVICE_CODE != 0
+        NV_IF_TARGET(NV_IS_DEVICE,
+        (
               if ((cub::LaneId() % LOGICAL_WARP_THREADS) >= num_valid)
               {
                   _CubLog("%s\n", "Invalid lane ID in cub::WrapperFunctor::operator()");
                   cub::ThreadTrap();
               }
-          #endif
-      }
+        ),
+        ());
```

Comment:
Else branch could be omitted in case it's empty: `NV_IF_TARGET(NV_IS_DEVICE, (/* code */))`.